Providers/Profile.swift

```diff
@@ -296,7 +296,7 @@ public class BrowserProfile: Profile {
         let commands = items.map { item in
             SyncCommand.fromShareItem(item, withAction: "displayURI")
         }
-        self.remoteClientsAndTabs.insertCommands(commands, forClients: clients)
+        self.remoteClientsAndTabs.insertCommands(commands, forClients: clients) >>> { self.syncManager.syncClients() }
```

Comment:
Do you mean here that if, when we sync, there are commands queued to execute on _our_ client then those commands will not be executed?